js/apps/system/_admin/aardvark/APP/frontend/js/routers/versionCheck.js

```diff
@@ -152,27 +72,19 @@
           $('#currentVersion').click(function () {
             showInterface(currentVersion, json);
           });
-        // isVersionCheckEnabled(showInterface.bind(window, currentVersion, json))
-        };
-
-        // TODO: append to url below
-        /*
-        var browserInfo = getInformation()
-        console.log(browserInfo)
-        console.log(encodeURIComponent(JSON.stringify(browserInfo)))
 
-        var osInfo = getOS()
-        console.log(osInfo)
-        */
+          // isVersionCheckEnabled(showInterface.bind(window, currentVersion, json))
+        };
 
         $.ajax({
           type: 'GET',
           async: true,
           crossDomain: true,
           timeout: 3000,
           dataType: 'jsonp',
-          url: 'https://www.arangodb.com/repositories/versions.php' +
-            '?jsonp=parseVersions&version=' + encodeURIComponent(currentVersion.toString()),
+          url: 'https://www.arangodb.com/versions.php' +
+            '?jsonp=parseVersions&version=' + encodeURIComponent(currentVersion.toString())
+            + '&hash=' + encodeURIComponent(data.hash + '-' + data.browser),
```

Comment:
Maybe also check if `data.hash` and `data.browser` are available. (only if that could be the case and it is not guaranteed)

Reply:
removed